### PR TITLE
[bn] Updated Content/bn/docs/setup/production environment/turnkey solutions.md

### PR DESCRIPTION
**Pull request by azaynul10**
Updated Content/bn/docs/setup/production environment/turnkey solutions.md in Bangla

Fixes Issue : #44993
Related PR : #45009
Tracking Issue: #31677

### CODE REVIEW

File: content/bn/docs/setup/production-environment/turnkey-solutions.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: আগে থেকে প্রস্তুত ক্লাউড সমাধান
```
Comment (asem-hamid) [inline content/bn/docs/setup/production-environment/turnkey-solutions.md:2]: টার্নকি ক্লাউড সলিউশন(Turnkey Cloud Solutions)

File: content/bn/docs/setup/production-environment/turnkey-solutions.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 content_type: ধারণা
```
Comment (asem-hamid) [inline content/bn/docs/setup/production-environment/turnkey-solutions.md:3]: concept

**Comment by asem-hamid** [comment]
maintain line breaks

image

image

**Comment by asem-hamid** [comment]
প্রোডাকশন

image

**Comment by asem-hamid** [comment]
maintain line breaks

image

image

File: content/bn/docs/setup/production-environment/turnkey-solutions.md (hunk 2 above)
Comment (azaynul10) [reply content/bn/docs/setup/production-environment/turnkey-solutions.md]: fixed it

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**